xml/Microsoft.Extensions.Caching.Memory/MemoryCacheOptions.xml

```diff
@@ -185,7 +185,7 @@
         <ReturnType>Microsoft.Extensions.Caching.Memory.MemoryCacheOptions</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets the default configured <typeparamref name="TOptions" /> instance.</summary>
```

Comment:
I'm not sure whether TOptions is meaningful here? Here's the preview: https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.caching.memory.memorydistributedcacheoptions?branch=pr-en-us-4107&view=dotnet-plat-ext-3.1#explicit-interface-implementations

Reply:
Also, the description for CompactOnMemoryPressure on line 131 doesn't seem correct.

Reply:
> Also, the description for CompactOnMemoryPressure on line 131 doesn't seem correct.

@gewarren which part do you think is off for CompactOnMemoryPressure?

Reply:
@maryamariyan It duplicates the description for ExpirationScanFrequency and doesn't seem to match the property name.

![image](https://user-images.githubusercontent.com/24882762/92037650-f5681280-ed26-11ea-821a-3bcbc31798f8.png)

Reply:
Seems like CompactOnMemoryPressure is already obsolete